src/renderer/components/+cluster-settings/status.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Spinner } from "../spinner";
+import { Cluster } from "../../../main/cluster";
+
+interface Props {
+  cluster: Cluster;
+}
+
+function headerToKey(header: string, suffix: string): string {
+  return header.replace(/\s+/, "-").toLowerCase() + "-" + suffix.replace(/\s+/, "-").toLowerCase();
+}
+
+export class Status extends React.Component<Props> {
+  renderStatusRows(): JSX.Element[] {
+    const { cluster } = this.props;
+
+    const rows: [string, React.ReactNode][] = [
+      ["Online Status", cluster.online ? "online" : `offline (${cluster.failureReason || "unknown reason"}`],
+      ["Distribution", cluster.distribution],
+      ["Kerbel Version", cluster.version],
+      ["API Address", cluster.apiUrl],
+    ];
+
+    if (cluster.nodes > 0) {
+      rows.push(["Nodes Count", cluster.nodes]);
+    }
+
+    return rows
+      .map(([header, value]) => [
+        <h5 key={headerToKey(header, "header")}>{header}</h5>,
```

Comment:
Using `headerToKey` looks redundant, value of `key` might be any uniq `string | number`.
Therefore `key={header + "some-fish"}` or even `key={index}` should be enough for this case.